internal/services/resource/resource_group_data_source.go

```diff
@@ -22,9 +23,10 @@ func dataSourceResourceGroup() *pluginsdk.Resource {
 		},
 
 		Schema: map[string]*pluginsdk.Schema{
-			"name":     commonschema.ResourceGroupNameForDataSource(),
-			"location": commonschema.LocationComputed(),
-			"tags":     tags.SchemaDataSource(),
+			"name":      commonschema.ResourceGroupNameForDataSource(),
+			"location":  commonschema.LocationComputed(),
+			"tags":      tags.SchemaDataSource(),
+			"managedBy": managedby.SchemaManagedByOptional(),
```

Comment:
Provider properties are all lowercase and snake cased, so can we rename this to `managed_by` and also in-line the schema. Additionally all properties in data sources that aren't required should only be set to Computed.
```suggestion
			"managed_by": {
				Type:     pluginsdk.TypeString, 
				Computed: true,
			},
```